docs/Settings.md

```diff
@@ -57,4 +57,19 @@ The settings can be saved to and loaded from an XML file using the following met
   fun loadStateFromFile(project: Project, filePath: String)
   ```
 
-These methods allow you to persist configuration data by serializing `MinimizationPluginState` to XML and then deserializing it back into the application.
\ No newline at end of file
+These methods allow you to persist configuration data by serializing `MinimizationPluginState` to XML and then deserializing it back into the application.
+
+### How to add new minimization stage in settings
+
+1. In [MinimizationPluginState][plugin-state] add your stage to ```defaultStages```
+2. In [AppSetnigsComponent][app-settings-component] 
+   1. initialize new boxes by analogy at Stage list
+   2. update getter and setter for varibale ```stages```
```

Comment:
If you have free time, I rather recommend making a step-by-step example-based guide like
```
Suppose we want to add a simple DD stage.
1. Let's make UI components: "Is enabled" checkbox and .....
```

But yea, it's still optional and require a lot of time. So, it's optional